src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -123,8 +116,13 @@ ucs_status_t uct_cuda_ipc_get_unique_index_for_uuid(int* idx,
             return UCS_ERR_NO_MEMORY;
         }
 
-        uct_cuda_ipc_accessible_cache_init(md->peer_accessible_cache + original_cache_size,
-                                           new_capacity - original_capacity);
+        buffer    = md->peer_accessible_cache + original_cache_size;
+        num_elems = new_capacity - original_capacity;
+
+        for (elem = 0; elem < num_elems; elem++) {
+            p = UCS_PTR_BYTE_OFFSET(buffer, elem * offset);
+            *p = UCS_TRY;
+        }
```

Comment:
for (i = original_cache_size; i < new_capacity ; ++i) {
md->peer_accessible_cache[i] = UCS_TRY;
}